app/code/Magento/Swatches/Block/Adminhtml/Attribute/Edit/Options/Visual.php

```diff
@@ -84,15 +84,15 @@ public function getJsonConfig()
      * Parse swatch labels for template
      *
      * @codeCoverageIgnore
-     * @param null $swatchStoreValue
-     * @return string
+     * @param array $swatchStoreValue
```

Comment:
It can still be `null`. Could you also squash changes into single commit and force push?